packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -81,13 +80,25 @@ Limits are applied on a per-recipient basis and vary by email type. The limit fo
 
 ### Per-user limits
 
-API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
+API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/), the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
 
 | Action and Okta API endpoint                                      | Per User limits (all orgs) |
 | ----------------------------------------------------------------- | -------------------------: |
 | **Authenticate the same user:**<br>`/api/v1/authn`                | 4 per second               |
 | **Generate or refresh an OAuth 2.0 token:**<br>`/oauth2/v1/token` | 4 per second               |
 
+SMS and Call factor endpoints also have a per-username rate limit:
+
+#### 30 second verification rate limits
+
+This rate limit applies to a user's attempt to send an SMS or Call enrollment or verification message to the same phone number. The rate limit is one SMS or Call challenge per device every 30 seconds.
+
+> **Note:** Okta round-robins between SMS providers with every resend request to help ensure delivery of SMS OTP across different carriers.
+
+#### Enrollment rate limit
```

Comment:
I think we should add the endpoints that will be rate limited to this section. (Similar to Per-user limits section but without the defaults)

Reply:
@shayekhazaeli-okta or like the ### Workforce license rate limit multiplier section that has a sub section called "List of endpoints"? @ericcarroll-okta

Reply:
I don't have a strong opinion here.  But I can easily get you the list of endpoints - just ping me.